link/src/main/java/com/stripe/android/link/ui/signup/SignUpViewModel.kt

```diff
@@ -261,15 +263,13 @@ internal class SignUpViewModel @Inject constructor(
         override fun <T : ViewModel> create(modelClass: Class<T>): T {
             injector.inject(this)
             return subComponentBuilderProvider.get()
-                .prefilledEmail(email)
-                .build().signUpViewModel as T
+                .build()
+                .signUpViewModel as T
```

Comment:
Now that `SignUpViewModel` doesn't take any extra parameters, this can be deleted, and the `SignUpViewModelSubcomponent` too.
The `lateinit var subComponentBuilderProvider` can be replaced by a `val signUpViewModel: SignUpViewModel` that will be directly injected in `injector.inject(this)` above. Similar to `VerificationViewModel`.